src/app/component/general/eco-news/eco-news.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { Title } from '@angular/platform-browser';
+
+@Component({
+  selector: 'app-eco-news',
+  templateUrl: './eco-news.component.html',
+  styleUrls: ['./eco-news.component.css']
+})
+export class EcoNewsComponent implements OnInit {
+  public getFilterArray: Array<string> = [];
+  public actualDate = new Date();
```

Comment:
delete it and on line 12 should be new Date().getFullYear()

Reply:
fixed